<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>zeus学生信息管理</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="pageUserInfo">
       <widget class="QWidget" name="widget" native="true">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>9</y>
          <width>300</width>
          <height>381</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>350</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <widget class="QWidget" name="widgetId" native="true">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>50</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="labelIdTitle">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>工号:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelIdContent">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QWidget" name="widgetClass" native="true">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>50</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="labelClassTitle">
              <property name="maximumSize">
               <size>
                <width>45</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>年级组:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelClassContent">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QWidget" name="widgetPhone" native="true">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>50</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="labelPhoneTitle">
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>电话:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelPhoneContent">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QWidget" name="widgetAccountType" native="true">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>50</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="labelAccountTypeTitle">
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>身份:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelAccountTypeContent">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>18</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>个人信息</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="pageMyClass">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="0" colspan="2">
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="0">
            <widget class="QWidget" name="widgetId_3" native="true">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>50</height>
              </size>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="labelIdTitle_3">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>教师:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelIdContent_3">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QWidget" name="widgetId_4" native="true">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>50</height>
              </size>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="labelIdTitle_4">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>教室:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelIdContent_4">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QWidget" name="widgetId_5" native="true">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>50</height>
              </size>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="labelIdTitle_5">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>学分:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelIdContent_5">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QWidget" name="widgetId_6" native="true">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>50</height>
              </size>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QLabel" name="labelIdTitle_6">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>时间:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelIdContent_6">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QWidget" name="widgetId_2" native="true">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>50</height>
              </size>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="labelIdTitle_2">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>名称:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelIdContent_2">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QWidget" name="widgetId_7" native="true">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>50</height>
              </size>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <widget class="QLabel" name="labelIdTitle_7">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>周次:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelIdContent_7">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QComboBox" name="comboBox"/>
        </item>
        <item row="2" column="0">
         <widget class="QWidget" name="widget_4" native="true">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="pushButton_7">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>评教</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_6">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>退选</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="pushButton_8">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>上课班级</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_9">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>考勤管理</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_10">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>成绩管理</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>758</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(220, 220, 220)</string>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuUser">
    <property name="title">
     <string>用户</string>
    </property>
    <addaction name="actionUserInfo"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuClass">
    <property name="title">
     <string>班级</string>
    </property>
    <addaction name="actionMyClass"/>
    <addaction name="actionChooseClass"/>
    <addaction name="actionCreateClass"/>
   </widget>
   <widget class="QMenu" name="menuLeave">
    <property name="title">
     <string>请假</string>
    </property>
    <addaction name="actionLeaveAsk"/>
    <addaction name="actionLaeveControl"/>
   </widget>
   <widget class="QMenu" name="menuAdmin">
    <property name="title">
     <string>管理员</string>
    </property>
    <widget class="QMenu" name="menuAdd">
     <property name="title">
      <string>添加</string>
     </property>
     <addaction name="actionAddTeacher"/>
     <addaction name="actionAddClass"/>
     <addaction name="actionAddGrand"/>
    </widget>
    <widget class="QMenu" name="menuDataControl">
     <property name="title">
      <string>数据管理</string>
     </property>
     <addaction name="actionAllClass"/>
     <addaction name="actionAllGrand"/>
     <addaction name="actionAllStudent"/>
     <addaction name="actionAllCause"/>
    </widget>
    <addaction name="actionStatusControl"/>
    <addaction name="menuAdd"/>
    <addaction name="menuDataControl"/>
   </widget>
   <addaction name="menuUser"/>
   <addaction name="menuClass"/>
   <addaction name="menuLeave"/>
   <addaction name="menuAdmin"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionUserInfo">
   <property name="text">
    <string>我的信息</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出登录</string>
   </property>
  </action>
  <action name="actionMyClass">
   <property name="text">
    <string>我的课程</string>
   </property>
  </action>
  <action name="actionChooseClass">
   <property name="text">
    <string>选择课程</string>
   </property>
  </action>
  <action name="actionCreateClass">
   <property name="text">
    <string>创建课程</string>
   </property>
  </action>
  <action name="actionLeaveAsk">
   <property name="text">
    <string>请假申请</string>
   </property>
  </action>
  <action name="actionLaeveControl">
   <property name="text">
    <string>请假审批</string>
   </property>
  </action>
  <action name="actionStatusControl">
   <property name="text">
    <string>状态管理</string>
   </property>
  </action>
  <action name="actionAddTeacher">
   <property name="text">
    <string>添加教师</string>
   </property>
  </action>
  <action name="actionAddClass">
   <property name="text">
    <string>添加班级</string>
   </property>
  </action>
  <action name="actionAddGrand">
   <property name="text">
    <string>添加年级组</string>
   </property>
  </action>
  <action name="actionAllClass">
   <property name="text">
    <string>班级</string>
   </property>
  </action>
  <action name="actionAllGrand">
   <property name="text">
    <string>年级</string>
   </property>
  </action>
  <action name="actionAllStudent">
   <property name="text">
    <string>学生/教师</string>
   </property>
  </action>
  <action name="actionAllCause">
   <property name="text">
    <string>课程</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
